docs/provider/mainnet-provider-setup.md

```diff
@@ -0,0 +1,707 @@
+---
+sidebar_position: 1
+slug: /mainnet-provider-setup
+title: Setup
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::warning
+
+With the launch of `lava-testnet-2`, aspects of our provider setup have changed. TLS Certificates are now a required part of Provider setup. [Lavavisor](/lavavisor) is now the recommended route for running the provider process.
```

Comment:
wrong chain-id.
mainnet chainid is: lava-mainnet-1

Reply:
solved in new commit
https://github.com/silentnoname/lavadocs/commit/d2d639ebcf4b8f5d49b21bfee37c605e20a3c345